src/pages/Recipes.js

```diff
@@ -1,14 +1,30 @@
 import { StyledHeader } from "./Home";
-import { useNavigate } from "react-router-dom";
-import { useContext } from "react";
+import { useState, useEffect, useContext } from "react";
 import { UserContext } from "../util/UserContext";
 import styled from "styled-components";
+import NavBar from "../components/NavBar";
+import RecipeCard from "../components/RecipeCard";
 
 export default function Recipes() {
-  const navigate = useNavigate();
   const { guestArray, setGuestArray } = useContext(UserContext);
   const selectedGuests = guestArray.filter((guest) => guest.selected);
   let intolerances = getIntolerances();
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  async function fetchData() {
+    const url = `https://api.edamam.com/api/recipes/v2?type=public&q=pasta&app_id=93fe14fb&app_key=515cbd62c9864645af6de1076420bdb9${intolerances
+      .map((item) => `&health=${item}`)
+      .join("")}`;
```

Comment:
This chaining is unfortunately very verbose. I don't understand what is happening here. Maybe you could break that step down with good variable names.

For example
```js
const healthParams = intolerances.map((item) => `&health=${item}`).join("")
const url = `https://api.edamam.com/api/recipes/v2?type=public&q=pasta&app_id=93fe14fb&app_key=515cbd62c9864645af6de1076420bdb9${healthParams }`;
```